src/transformers/models/bort/modeling_bort.py

```diff
@@ -0,0 +1,231 @@
+# coding=utf-8
+# Copyright 2020, The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PyTorch BORT model. """
+
+from ...file_utils import add_start_docstrings
+from ...utils import logging
+from ..bert.modeling_bert import (
+    BertForMaskedLM,
+    BertForMultipleChoice,
+    BertForQuestionAnswering,
+    BertForSequenceClassification,
+    BertForTokenClassification,
+    BertModel,
+)
+from .configuration_bort import BortConfig
+
+
+logger = logging.get_logger(__name__)
+
+_TOKENIZER_FOR_DOC = "RobertaTokenizer"
+
+BORT_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    # See all BORT models at https://huggingface.co/models?filter=bort
+]
+
+BORT_START_DOCSTRING = r"""
+
+    This model inherits from :class:`~transformers.PreTrainedModel`. Check the superclass documentation for the generic
+    methods the library implements for all its model (such as downloading or saving, resizing the input embeddings,
+    pruning heads etc.)
+
+    This model is also a PyTorch `torch.nn.Module <https://pytorch.org/docs/stable/nn.html#torch.nn.Module>`__
+    subclass. Use it as a regular PyTorch Module and refer to the PyTorch documentation for all matter related to
+    general usage and behavior.
+
+    Parameters:
+        config (:class:`~transformers.BortConfig`): Model configuration class with all the parameters of the
+            model. Initializing with a config file does not load the weights associated with the model, only the
+            configuration. Check out the :meth:`~transformers.PreTrainedModel.from_pretrained` method to load the model
+            weights.
+"""
+
+
+@add_start_docstrings(
+    "The bare BORT Model transformer outputting raw hidden-states without any specific head on top.",
+    BORT_START_DOCSTRING,
+)
+class BortModel(BertModel):
+    """
+    This class overrides :class:`~transformers.BertModel`. Please check the superclass for the appropriate
+    documentation alongside usage examples.
+
+    Examples::
+        >>> from transformers import BortModel, BortTokenizer
+
+        >>> model = BortModel.from_pretrained("bort")
+        >>> tokenizer = BortTokenizer.from_pretrained("bort")
+
+        >>> inputs = tokenizer.encode_plus("The eastern endpoint of the canal is the Hubertusbrunnen.", return_tensors="pt")
+        >>> outputs = model(**inputs)
+        >>> hidden_states = outputs.last_hidden_state
+    """
+
+    config_class = BortConfig
```

Comment:
we should add `model_type = 'bort'` for each class here -> see MT5 for comparison: https://github.com/huggingface/transformers/blob/51adb97cd644a5840d971868d18c1d436fd6ff5d/src/transformers/models/mt5/modeling_mt5.py#L43